### PR TITLE
navbar

### LINKED ISSUE #6: Navbar Component
## Expected Behaviour

Create a navbar component that shows 4 icons that represent the base pages (Home, List, Bookmarks, Account)
You can use the Heroicons package
Each icon on the navbar can have a active state

## Checklist

_Detail the steps below to help us reproduce the problem you encountered._

- [x] Follow the component design in figma https://www.figma.com/design/TOCJzJFMXE7PL1dQyGWqXZ/midterm?node-id=193-51&t=3bsn1O4cipxo3ue9-1
- [x] Show the active state on one icon
- [x] Everything is static for now

## Hints

- https://www.npmjs.com/package/@heroicons/react

### PR DESCRIPTION
**Pull request by multimediaschneider**
## Checklist

Confirm you have completed the following actions prior to submitting this PR.

- [x] There is an existing issue report for this PR.
- [x] I have created a feature branch.
- [x] My changes have been committed.
- [x] I have pushed my changes to the branch.

## Title

Navbar component created

## Issue Resolution

_Tell us which issue this PR fixes._

Closes #6

## Proposed Changes

_List your proposed changes below._

-
-
-

## New or Changed Features

_Does this PR provide new or changed features or enhancements? If so, what is included in this PR?_

-
-
-

### CODE REVIEW

File: src/layouts/.gitkeep
Comment (dnmct) [inline src/layouts/.gitkeep:1]: Pls don't remove this if you don't add a file in this folder

Comment (dnmct) [inline src/layouts/.gitkeep:1]: Pls don't remove this if you don't add a file in this folder

File: src/components/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  QueueListIcon,
+  UserIcon,
+} from "@heroicons/react/24/solid";
+
+export function NavBar() {
+  const [icon, setIconState] = useState("home");
+
+  return (
+    <div className="bg-dark flex pl-16 pr-16 pt-8 pb-8 gap-6 justify-evenly items-center">
+      <HomeIcon
```
Comment (dnmct) [inline src/components/NavBar.tsx]: You could create an array of navItems and map over it instead of repeating the element

**Review by dnmct — CHANGES_REQUESTED** [review]

File: src/layouts/.gitkeep
Comment (multimediaschneider) [reply src/layouts/.gitkeep:1]: sorry

Comment (multimediaschneider) [reply src/layouts/.gitkeep:1]: kind of what i thought

**Review by dnmct — APPROVED** [review]

**Pull request merged**